examples/middleware/router.yaml

```diff
@@ -2,6 +2,9 @@ sandbox:
   enabled: true
 supergraph:
   introspection: true
+  listen: 0.0.0.0:80
```

Comment:
try to avoid using 80 port it will fail with an error `listen tcp :80: bind: permission denied`